package.json

```diff
@@ -25,8 +25,8 @@
     "url": "https://github.com/astarte-platform/astarte-dashboard"
   },
   "scripts": {
-    "start": "PORT=8080 react-scripts start",
-    "build": "react-scripts build",
+    "start": "PORT=8080 react-scripts --openssl-legacy-provider start",
+    "build": "react-scripts --openssl-legacy-provider build",
```

Comment:
This is bit ugly as a workaround, but i think it's fine for now. We will eventually upgrade our build system so it doesn't rely on react-scripts. What do you say @davidebriani ?

Reply:
Yes it's fine by me too, we will fix that issue in a later PR